biothings/hub/autoupdate/uploader.py

```diff
@@ -207,48 +210,110 @@ async def restore_snapshot(self, build_meta, job_manager, **kwargs):
 
         # repository is now ready, let's trigger the restore
         snapshot_name = build_meta["metadata"]["snapshot_name"]
+
+        # backup the original value of indexer's replica
+        original_number_of_replicas = idxr.get_internal_number_of_replicas() or 0
+
+        alias_name = idxr.canonical_index_name
+        d = datetime.datetime.now().strftime('%Y%m%d')
+        base_index_name = f'{alias_name}_{snapshot_name}_{d}_'
```

Comment:
@sengineer0 can we set the default `base_index_name` just be the `snapshot_name`? `snapshot_name` was typically created as a unique name (with timestamp and random string). Typically, we don't need to restore the same snapshot twice.

But I can see this is useful when testing, so you can restore the same snapshot multiple times, without manually removing the existing index. Maybe you can leave it as an option to append an additional string to the `base_index_name`. But in this case, I would like just do `f'{snapshot_name}_{ts}'` with `ts` as the timestamp up to the minute, so that it will be unique enough. This can be turned on with `kwargs['append_ts']==True` parameter.

Reply:
ok sure. I will update as your recommendation. Thanks.